framework/src/media/utils/MediaUtils.cpp

```diff
@@ -54,6 +59,48 @@ void toUpperString(std::string &str)
 	}
 }
 
+audio_container_t getAudioContainerFromPath(std::string datapath)
```

Comment:
It should return with file extension.
It needs to consider many abnormal case in the future. for example, no extension, wrong extension.

Reply:
You are right, current code does not work well in such abnormal cases.
I will keep in mind and refine it in future.
Thanks!